### PR TITLE
Add position modal

### PR DESCRIPTION
**Pull request by pratikshamandlik99**

### CODE REVIEW

File: ats-web/src/components/shared/shared.css [hunk 1]
```diff
@@ -2,4 +2,4 @@
     display: flex;
     justify-content: center;
     margin: 10px;
-}
\ No newline at end of file
+}
```
Comment (aparna-salokhe) [inline ats-web/src/components/shared/shared.css]: Remove this file and add required styles to 'ats-web/src/styles/shared.css' file.

File: ats-web/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,15553 @@
+{
```
Comment (aparna-salokhe) [inline ats-web/package-lock.json]: Exclude this file too

File: ats-web/package.json [hunk 3]
```diff
@@ -1,37 +0,0 @@
-{
```
Comment (aparna-salokhe) [inline ats-web/package.json]: we need this file package.json.
package-lock.json should not be included in PR

**Comment by pratikshamandlik99** [comment]
merge add-position-modal

**Pull request merged**